docs/build/javascript/quick-start.md

```diff
@@ -0,0 +1,133 @@
+# Quickstart
+Learn the basics by setting up and interacting with the [Ceramic Javascript Client](./javascript/installation.md).
+
+!!! warning ""
+    **Want an even faster way to try Ceramic? Visit the [Playground Demo App](https://playground.ceramic.dev){:target="_blank"} to test the full stack of Ceramic components in the browser.
+
+## **1. Install the Client**
+
+Visit the [Ceramic Client](./javascript/installation.md) page for instructions on how to quickly install the Client.
+
+## **2. Create a stream**
+Creating streams is dependent on the StreamType. More details can be found in our [StreamTypes Overview](../streamtypes/overview.md). A basic sample using the [TileDocument StreamType](../streamtypes/tile-document/overview.md) can be found below.
+
+=== "Command"
+
+    ```JavaScript
+    import { TileDocument } from '@ceramicnetwork/stream-tile'
+
+    const doc = await TileDocument.create(ceramic, {hello: 'world'})
+
+    const streamId = doc.id.toString()
```

Comment:
Maybe also add a `console.log(doc.content)` here to quickly show where the data end up?